frontend/src/hooks/useAdminCounts.js

```diff
@@ -0,0 +1,14 @@
+import { useQuery } from "@tanstack/react-query";
+
+import { PROJECT_ID } from "../store";
+import { getConfig } from "../utils/fetch_configs";
+
+const useAdminCounts = () =>
+    useQuery({
+        queryKey: ["adminCounts", PROJECT_ID],
+        queryFn: () =>
+            fetch(`/api/data_admin_counts/${PROJECT_ID}/`, getConfig())
```

Comment:
Having a new issue when trying to load the history page. It seems to be bouncing back and forth between data_admin_counts and get_label_history. My guess is that they each call each other

```
backend              | [04/Aug/2023 14:33:03] "GET /api/get_label_history/3/?page=1&unlabeled=false HTTP/1.1" 200 48
backend              | [04/Aug/2023 14:33:13] "GET /api/data_admin_counts/3/ HTTP/1.1" 200 27
backend              | [04/Aug/2023 14:33:13] "GET /api/get_label_history/3/?page=1&unlabeled=false HTTP/1.1" 200 48
backend              | [04/Aug/2023 14:33:15] "GET /api/data_admin_counts/3/ HTTP/1.1" 200 27
backend              | [04/Aug/2023 14:33:15] "GET /api/get_label_history/3/?page=1&unlabeled=false HTTP/1.1" 200 48
backend              | [04/Aug/2023 14:33:30] "GET /api/data_admin_counts/3/ HTTP/1.1" 200 27
backend              | [04/Aug/2023 14:33:30] "GET /api/get_label_history/3/?page=1&unlabeled=false HTTP/1.1" 200 48
backend              | [04/Aug/2023 14:33:42] "GET /api/data_admin_counts/3/ HTTP/1.1" 200 27
backend              | [04/Aug/2023 14:33:42] "GET /api/get_label_history/3/?page=1&unlabeled=false HTTP/1.1" 200 48
backend              | [04/Aug/2023 14:34:08] "GET /api/data_admin_counts/3/ HTTP/1.1" 200 27
backend              | [04/Aug/2023 14:34:08] "GET /api/get_label_history/3/?page=1&unlabeled=false HTTP/1.1" 200 48
backend              | [04/Aug/2023 14:34:10] "GET /api/data_admin_counts/3/ HTTP/1.1" 200 27
backend              | [04/Aug/2023 14:34:10] "GET /api/get_label_history/3/?page=1&unlabeled=false HTTP/1.1" 200 48
```